oss_fuzz_integration/README.md

```diff
@@ -29,7 +29,7 @@ From within this directory, run the commands:
 
 # Test a project
 cd oss-fuzz
-../run_both.sh htslib 20
+python3 ../manager.py full htslib 20
```

Comment:
nit: should we call this `runner.py`? and the options be like `runner.py introspector` vs. `runner.py coverage`?

I think this will be more intuitive, as this script is about running OSS-fuzz integration rather managing things. WDYT?

Reply:
> nit: should we call this `runner.py`? and the options be like `runner.py introspector` vs. `runner.py coverage`?

I'm happy to either way! one issue to overcome in that context is to avoid confusion in that `introspector` e.g. does both (1) build with asan; (2) run using asan fuzzers; (3) build coverage; (4) run coverage and finally (5) run introspector -- as such calling it `introspector` may be confusing? I called it `full` atm for that reason, although admittedly it's not super descriptive.

Reply:
> to avoid confusion in that `introspector` e.g. does

I wouldn't worry much about it honestly. I think the user should know the implicit meaning of running `introspector` includes running fuzzer and coverage. Such implicitly is already in `coverage` run, i.e we don't worry to mention that running for coverage needs the fuzzer be run before and corpus be available.